samples/Calculator/README.md

```diff
@@ -0,0 +1,5 @@
+**Scenarios**
```

Comment:
Should we briefly explain how the solution is composed?

Reply:
added